diff --git a/Gemfile b/Gemfile
index 9b088942..2c77a011 100644
--- a/Gemfile
+++ b/Gemfile
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '>=3.2.4'
+ruby '>=3.2.0'
 
 # Ensure github repositories are fetched using HTTPS
 git_source(:github) do |repo_name|
@@ -118,7 +118,7 @@ gem 'loofah', '~> 2.22'
 gem 'mail', '>= 2.8.1'
 gem 'mini_magick', ">= 4.12.0"
 gem 'multi_xml'
-gem "nokogiri", ">= 1.16.6"
+gem "nokogiri", force_ruby_platform: true
 gem 'omniauth'
 gem 'rails', '~> 6.1.7', '>= 6.1.7.8'
 gem 'rails-html-sanitizer', '~> 1.6'
